internal/util/vcs/git/remote_svc_provider/github_svc_provider.go

```diff
@@ -28,8 +28,8 @@ type GithubSvcProvider interface {
 }
 
 // ValidateToken creates a new instance of GithubSvcProvider.
-func (g *githubSvcProvider) ValidateToken(repoUrl string, repoIdentifier string) (string, error) {
-	if _, err := g.git.GetGitToken(g.git.GetRemoteServiceProvider(repoUrl)); err != nil {
+func (g *githubSvcProvider) ValidateToken(repoUrl string, repoIdentifier string, tokenFromProvider string) (string, error) {
+	if _, err := g.git.GetGitToken(g.git.GetRemoteServiceProvider(repoUrl), tokenFromProvider); err != nil {
 		suggestions, gitTokenErr := gitTokenErrorHandler(err)
```

Comment:
Assigned variables can be removed.
```suggestion
		return gitTokenErrorHandler(err)
```